src/uwtools/drivers/fv3.py

```diff
@@ -79,9 +80,21 @@ def files_copied(self):
         Files copied for run.
         """
         yield self.taskname("files copied")
+        yield [Copier(config=self.config.get("files_to_copy", {}), target_dir=self.rundir).go()]
```

Comment:
This got a bit mangled in my GitHub view, but this replaces: 

```
            filecopy(src=Path(src), dst=self.rundir / dst)
            for dst, src in self.config.get("files_to_copy", {}).items()
```

And is similarly done for the Linker, and the new hardlinked call to the Linker.